base_installation.sh

```diff
@@ -9,109 +9,90 @@ PATH="${PATH}:${HOME}/.local/bin"
 mkdir -p downloads
 mkdir -p software
 mkdir -p cache
-echo "Installing pip.\n"
+
+echo -e "\nInstalling pip.\n"
 if [ ! `which pip` ] ; then
   cd downloads
   wget "https://bootstrap.pypa.io/get-pip.py"
   # install pip locally
   python get-pip.py --user --ignore-installed
   cd ..
+else
+    echo "\nPip is already installed.\n"
```

Comment:
@KyleBehiels If you want the "\n" (new line characters) to be executed, you need to use ```echo -e```

Reply:
@KyleBehiels Did you get this.

Reply:
Yes sorry I just missed a few